waiter/bin/ci/run-integration-tests.sh

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
-# Usage: run-integration-tests.sh [TEST_SELECTOR]
+# Usage: run-integration-tests.sh [TEST_SELECTOR] [TEST_COMMAND]
```

Comment:
Nitpick: I prefer `run-integration-tests.sh [TEST_COMMAND] [TEST_SELECTOR]` because that's the order we use with lein.